### PR TITLE
AST-2944 - fix: BBPress Compatibility with Astra Theme

### PR DESCRIPTION
**Pull request by yssambare12**
**Test Data**: After installing BBpress the author bio page showing multiple time check screenshot for more details. 

**Steps To Reproduce:** 

All is explained in this video: https://share.getcloudapp.com/p9uLY1Zr 

**Any Other details:** 

https://secure.helpscout.net/conversation/2129150514/296764?folderId=4660463
https://secure.helpscout.net/conversation/2140044270/302332

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [x] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/markup-extras.php [hunk 1]
```diff
@@ -1688,3 +1688,21 @@ function astra_skip_elementor_onboarding( $network_wide ) {
 }
 
 add_action( 'activate_elementor/elementor.php', 'astra_skip_elementor_onboarding' );
+
+
+/**
+ * BBPress Multiple user profile issue compatibility.
```
Comment (yssambare12) [inline inc/markup-extras.php:1694]: * BBPress Multiple user profile compatibility issue.

File: inc/markup-extras.php [hunk 2]
```diff
@@ -1688,3 +1688,26 @@ function astra_skip_elementor_onboarding( $network_wide ) {
...
+ */
+function astra_bbpress_issue( $value ) {
+	if ( class_exists( 'bbpress' ) ) {
+		if ( bbp_is_single_user() ) { // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			return false;
+		}
+		if ( bbp_is_search() ) { // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			return false;
+		}
+	}
```
Comment (imnavanath) [inline inc/markup-extras.php:1708]:
```suggestion
	if ( class_exists( 'bbpress' ) && ( bbp_is_single_user() || bbp_is_search() ) ) {
			return false;
	}
```

File: changelog.txt [hunk 3]
```diff
@@ -1,5 +1,6 @@
 v4.1.1 (Unreleased)
 - Fix: Customizer - Typography and Button Preset options selected preset not highlighting and reset button not functioning.
+- Fix: Showing author info multiple time and goes to infinite loop.
```
Comment (imnavanath) [inline changelog.txt:3]:
```suggestion
- Fix: Author Info box is visible multiple times and goes into infinite loop in case of BBPress.
```

**Review by imnavanath — APPROVED** [review]

File: inc/markup-extras.php (hunk 2 above)
Comment (yssambare12) [reply inc/markup-extras.php]: Updated.

File: changelog.txt (hunk 3 above)
Comment (yssambare12) [reply changelog.txt]: Updated

File: inc/markup-extras.php [hunk 4]
```diff
@@ -1688,3 +1688,23 @@ function astra_skip_elementor_onboarding( $network_wide ) {
...
+ * BBPress Multiple user profile compatibility issue.
+ *
+ * @param bool $value For checking this issue is still persist or not.
+ *
+ * @since x.x.x
+ */
+function astra_bbpress_issue( $value ) {
+	/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
+	if ( class_exists( 'bbpress' ) && ( bbp_is_single_user() || bbp_is_search() ) ) {
+		/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
```
Comment (imnavanath) [inline inc/markup-extras.php:1703]:
```suggestion
	/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
	/** @psalm-suppress UndefinedFunction  */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
	if ( class_exists( 'bbpress' ) && ( bbp_is_single_user() || bbp_is_search() ) ) {
		/** @psalm-suppress UndefinedFunction  */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
		/** @psalm-suppress InvalidArgument */ // phpcs:ignore Generic.Commenting.DocComment.MissingShort
```

**Pull request merged**